examples/ex-batched-matrix-multiply.cpp

```diff
@@ -27,299 +27,515 @@
-        Cl2view(i,1,1) = Al2view(i,1,0)*Bl2view(i,0,1) + Al2view(i,1,1)*Bl2view(i,1,1) + Al2view(i,1,2)*Bl2view(i,2,1);
-        Cl2view(i,1,2) = Al2view(i,1,0)*Bl2view(i,0,2) + Al2view(i,1,1)*Bl2view(i,1,2) + Al2view(i,1,2)*Bl2view(i,2,2);
-        
-        Cl2view(i,2,0) = Al2view(i,2,0)*Bl2view(i,0,0) + Al2view(i,2,1)*Bl2view(i,1,0) + Al2view(i,2,2)*Bl2view(i,2,0);
-        Cl2view(i,2,1) = Al2view(i,2,0)*Bl2view(i,0,1) + Al2view(i,2,1)*Bl2view(i,1,1) + Al2view(i,2,2)*Bl2view(i,2,1);
-        Cl2view(i,2,2) = Al2view(i,2,0)*Bl2view(i,0,2) + Al2view(i,2,1)*Bl2view(i,1,2) + Al2view(i,2,2)*Bl2view(i,2,2);
-        
-      });
+    RAJA::forall<RAJA::omp_parallel_for_exec>(
+        RAJA::RangeSegment(0, N), [=](Index_type e) {
+
+          Cview2(e, 0, 0) = Aview2(e, 0, 0) * Bview2(e, 0, 0)
+                            + Aview2(e, 0, 1) * Bview2(e, 1, 0)
+                            + Aview2(e, 0, 2) * Bview2(e, 2, 0);
+          Cview2(e, 0, 1) = Aview2(e, 0, 0) * Bview2(e, 0, 1)
+                            + Aview2(e, 0, 1) * Bview2(e, 1, 1)
+                            + Aview2(e, 0, 2) * Bview2(e, 2, 1);
+          Cview2(e, 0, 2) = Aview2(e, 0, 0) * Bview2(e, 0, 2)
+                            + Aview2(e, 0, 1) * Bview2(e, 1, 2)
+                            + Aview2(e, 0, 2) * Bview2(e, 2, 2);
+
+          Cview2(e, 1, 0) = Aview2(e, 1, 0) * Bview2(e, 0, 0)
+                            + Aview2(e, 1, 1) * Bview2(e, 1, 0)
+                            + Aview2(e, 1, 2) * Bview2(e, 2, 0);
+          Cview2(e, 1, 1) = Aview2(e, 1, 0) * Bview2(e, 0, 1)
+                            + Aview2(e, 1, 1) * Bview2(e, 1, 1)
+                            + Aview2(e, 1, 2) * Bview2(e, 2, 1);
+          Cview2(e, 1, 2) = Aview2(e, 1, 0) * Bview2(e, 0, 2)
+                            + Aview2(e, 1, 1) * Bview2(e, 1, 2)
+                            + Aview2(e, 1, 2) * Bview2(e, 2, 2);
+
+          Cview2(e, 2, 0) = Aview2(e, 2, 0) * Bview2(e, 0, 0)
+                            + Aview2(e, 2, 1) * Bview2(e, 1, 0)
+                            + Aview2(e, 2, 2) * Bview2(e, 2, 0);
+          Cview2(e, 2, 1) = Aview2(e, 2, 0) * Bview2(e, 0, 1)
+                            + Aview2(e, 2, 1) * Bview2(e, 1, 1)
+                            + Aview2(e, 2, 2) * Bview2(e, 2, 1);
+          Cview2(e, 2, 2) = Aview2(e, 2, 0) * Bview2(e, 0, 2)
+                            + Aview2(e, 2, 1) * Bview2(e, 1, 2)
+                            + Aview2(e, 2, 2) * Bview2(e, 2, 2);
+
+        });
     timer.stop();
-    
+
     RAJA::Timer::ElapsedType tMin = timer.elapsed();
-    if(tMin < myMin) myMin = tMin;
+    if (tMin < minRun) minRun = tMin;
     timer.reset();
   }
-  std::cout<<"Matrix Multiplication with layout 2 on CPU run time : "<<myMin<<" seconds"<<std::endl;
-  //---------------------------------------------
-  
-//
-//Compare output
-//
-  compareOutput(Cview, Cl2view, NMAT);
+  std::cout << "Performing batched matrix multiplication with layout 2 using "
```

Comment:
Please put the print message about which example is run before the example is run (same comment as above.